print_companions.sql

```diff
@@ -0,0 +1,36 @@
+CREATE FUNCTION fnCompanions
+(
+	@EpisodeId AS int
+)
+RETURNS VARCHAR(MAX) AS
+BEGIN
+DECLARE @Output AS VARCHAR(2048) =''
+DECLARE @CompanionName AS varchar(64)
+Declare CompanionCursor Cursor FOR
```

Comment:
cursors in sql has a really bad performance, 
can you try to write it using SQL string aggregate functions and compare the performance please.